.github/workflows/ci.yml

```diff
@@ -39,6 +39,9 @@ jobs:
       - name: Tests
         run: bundle exec sus
 
+      - name: GreenDots tests
+        run: bundle exec gd gd
```

Comment:
We need the extra `gd` here for the test directory, since `test` is taken by Sus.